client/extensions/woocommerce/app/dashboard/index.js

```diff
@@ -3,30 +3,67 @@
  */
 import React, { Component, PropTypes } from 'react';
 import classNames from 'classnames';
+import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import Main from 'components/main';
 import Card from 'components/card';
-import SectionHeader from 'components/section-header';
+import { getSelectedSite } from 'state/ui/selectors';
+import Main from 'components/main';
+import Setup from './setup';
 
-export default class Dashboard extends Component {
+class Dashboard extends Component {
 
 	static propTypes = {
 		className: PropTypes.string,
 	};
 
-	render() {
+	onStoreSetupFinished = () => {
+		// TODO - save that setup has been finished to the store's state on WPCOM
+		// TODO - is there a way to set an option on the store site?
+	}
+
+	renderStoreSetup = () => {
+		const { selectedSite } = this.props;
+		return (
+			<Setup
+				onFinished={ this.onStoreSetupFinished }
+				site={ selectedSite }
+			/>
+		);
+	}
+
+	renderStoreManagement = () => {
+		return (
+			<Card>
+				<p>This is the start of something great!</p>
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14202/commits/e14d133c34678fb404ae593d8f802590f4c2b3d9